lib/utils/cli.go

```diff
@@ -466,60 +466,60 @@ const defaultCommandPrintfWidth = 12
 
 // defaultUsageTemplate is a fmt format that defines the usage template with
 // compactly formatted commands. Should be only used in createUsageTemplate.
-const defaultUsageTemplate = `{{define "FormatCommand"}}\
-{{if .FlagSummary}} {{.FlagSummary}}{{end}}\
-{{range .Args}} {{if not .Required}}[{{end}}<{{.Name}}>{{if .Value|IsCumulative}}...{{end}}{{if not .Required}}]{{end}}{{end}}\
-{{end}}\
-
-{{define "FormatCommands"}}\
-{{range .FlattenedCommands}}\
-{{if not .Hidden}}\
-  {{.FullCommand | printf "%%-%ds"}}{{if .Default}} (Default){{end}} {{ .Help }}
-{{end}}\
-{{end}}\
-{{end}}\
-
-{{define "FormatUsage"}}\
+const defaultUsageTemplate = `{{define "FormatCommand" -}}
```

Comment:
Is this the only template affected? Should we chase all templates used in tsh?

Reply:
Do we have any other templates? If I am correct, this is the only custom template we have.